develop/CICD/Azure DevOps/CICD_Azure_Devops_Examples.md

```diff
@@ -0,0 +1,54 @@
+---
+uid: CICD_Azure_DevOps_Examples
+---
+
+# Azure DevOps CI/CD Examples
+
+## Basic Deployment Example
+
+A basic pipeline for upload to catalog and/or deployment to cloud-connected agents.
+It's recommended to combine this with quality control beforehand such as executing static code analysis and running tests.
+
+## Creating a dataminer.services key

```

Comment:
Should "Creating a dataminer.services key" be considered part of the "basic deployment example"? Because right now, they're the same header level, so it looks as if this is a completely new section unrelated to what came before.

Reply:
Yes, I changed this to be part of the basic deployment example